spec/features/user_registration_spec.rb

```diff
@@ -1,13 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe "Users", type: :feature do
-  it "can sign up" do
+
```

Comment:
Rubocop does not like this empty line.

Reply:
If you disagree with RuboCop, feel free to edit its config. Tools should not burden you!

Reply:
I like RuboCop for asserting a common coding style. Do we have any format to discuss questions like this in the large group?

Reply:
Maybe propose changes you'd like to make to the `.rubocop.yml` in a PR? Then that could be used for discussion on concrete changes

Reply:
For example, I would probably disable/modify the rules that lead to this: 
![image](https://user-images.githubusercontent.com/1652117/101173801-51eb0e80-3643-11eb-8d9c-be51440f1de2.png)

Reply:
I would suggest the following for `.rubocop.yml`. Empty lines might be nice for structuring tests?

```Ruby
Layout/EmptyLinesAroundBlockBody:
  Exclude:
    - 'spec/**/*.rb'
```